src/components/tasks/card/index.tsx

```diff
@@ -199,24 +188,34 @@ const Card: FC<CardProps> = ({
         }
     }
 
-    function renderDate(fromNowEndsOn: string, isEditable: boolean) {
+    function renderDate(date: Date, isEditable: boolean, type: any) {
+        const formattedDate = moment(date).format('MMM D, YYYY');
         if (isEditable) {
             return (
                 <input
                     type="date"
                     onChange={(e) => {
                         setDateTimes(e.target.value);
-                        handelDateChange(e, 'endsOn');
+                        handelDateChange(e, type);
                     }}
                     value={dateTimes}
                     data-testid="date"
                 />
             );
         }
         return (
-            <span className={styles.cardStrongFont} role="button" tabIndex={0}>
-                {!cardDetails.endsOn ? 'TBD' : fromNowEndsOn}
-            </span>
+            <>
```

Comment:
fragment not needed

Reply:
div was creating an issue here, in styling.

Reply:
I meant to say fragment is also not needed here as we have single single span element